libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/widgets/AutoResizeTextView.java

```diff
@@ -16,7 +17,7 @@
  * See http://stackoverflow.com/a/5535672
  *
  */
-public class AutoResizeTextView extends TextView {
+public class AutoResizeTextView extends AppCompatTextView {
```

Comment:
Just wondering, can you share some background about this change? Is it some lint related warning or something else? Thanks!